website/src/word-panel.tsx

```diff
@@ -137,7 +137,11 @@ const CollapsiblePanel = (p: {
   const disclosure = useDisclosureState({ visible: true })
   return (
     <div className={css.collPanel} aria-label={p.title}>
-      <Disclosure {...disclosure} className={css.collPanelButton}>
+      <Disclosure
+        {...disclosure}
+        className={css.collPanelButton}
+        aria-label={`${p.title} header`}
```

Comment:
Let's just use `p.title` here, screen readers will already get an indication that this is a button which opens more details.

Reply:
That feels redundant though since the whole element, the full collapsible panel, already has an aria-label that is p.title. Would we have to remove that label for the whole panel or change it to something else?

Reply:
Oh I'm sorry, I didn't see that! I think that common practice is to label the `Disclosure` button and not label the div we are putting around it.

Reply:
Ah! okay.